Content.Shared/Chemistry/SharedChemMaster.cs

```diff
@@ -148,18 +140,40 @@ public sealed class ChemMasterBoundUserInterfaceState : BoundUserInterfaceState
 
         public readonly bool UpdateLabel;
 
-        public ChemMasterBoundUserInterfaceState(
-            ContainerInfo? inputContainerInfo, ContainerInfo? outputContainerInfo,
-            IReadOnlyList<ReagentQuantity> bufferReagents, FixedPoint2 bufferCurrentVolume,
-            uint selectedPillType, uint pillDosageLimit, bool updateLabel)
+        /// <summary>
+        /// A list of the reagents and their amounts within the pill buffer, if applicable.
+        /// </summary>
+        public readonly IReadOnlyList<ReagentQuantity> PillBufferReagents; // DeltaV
+
+        public readonly int SortMethod; // DeltaV
+        public readonly int TransferringAmount; // DeltaV
+
+        public readonly FixedPoint2? PillBufferCurrentVolume; // DeltaV
+
+        public ChemMasterBoundUserInterfaceState(  // DeltaV
+            ContainerInfo? inputContainerInfo,
+            IReadOnlyList<ReagentQuantity> bufferReagents,
+            IReadOnlyList<ReagentQuantity> pillBufferReagents,
+            FixedPoint2 bufferCurrentVolume,
+            FixedPoint2 pillBufferCurrentVolume,
+            uint selectedPillType,
+            uint pillDosageLimit,
+            bool updateLabel,
+            int sortMethod,
+            int transferringAmount)
```

Comment:
this still reformats the constructor, keep it how it was and move all deltav fields to the end